### DISCUSSION TITLE
Does Pyarmor 8.4 support FreeBSD OS?

### DISCUSSION THREAD

**Discussion by Soujanya459** [dashingsoft/pyarmor#1554] [General]
Hi Jondy,

We are trying to upgrade python version to 3.12 and also pyarmor 8.4 in our product..

I see Pyarmor 8.3 supports FreeBSD.. how about 8.4... ? 
Does all 8+ versions support FreeBSD right, you will not miss it ? 

ThankYou,
Soujanya

**Comment by jondy** [comment]
Yes.
